lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -148,6 +148,33 @@ class ReportRenderer {
   }
 
   /**
+   * @param {!ReportRenderer.CategoryJSON} category
+   * @return {!DocumentFragment}
+   */
+  _renderScoreGauge(category) {
```

Comment:
I'm not really in love with a bunch of isolated components filling `report-renderer.js`, but there's not really an obvious place for them to go now that details seems to not be what I thought it was haha